### ISSUE TITLE
Is this project related to LibShortText?

### ISSUE THREAD

**Issue by huntzhan** [2shou/TextGrocery#16]
The numbers are identical!

http://guoze.me/2014/09/25/libshorttext-introduction/

scikit-learn(nb) 	76.8% 	134
scikit-learn(svm) 	76.9% 	121
libshorttext 	79.6% 	49

Classifier 	Accuracy 	Time cost(s)
scikit-learn(nb) 	76.8% 	134
scikit-learn(svm) 	76.9% 	121
TextGrocery 	79.6% 	49

**Comment by suntxy** [comment]
Yes it's based on TextGrocery, and added library jieba for chinese tokenize.
